app/views/programs/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container d-flex justify-content-between my-3">
+  <h2 class="mb-0">Programs</h2>
+  <%= link_to new_program_path, class: "btn btn-outline-secondary rounded-3 py-1 px-2 fs-1.5 link-underline link-underline-opacity-0" do %>
```

Comment:
Mentioned to you, but wasn't sure if the user should be able to create programs or if that should be more of an admin thing

Reply:
Added issue #2 to address this later